src/css/lib/monster.scss

```
@@ -685,80 +685,79 @@ label[class*='monster-radio'].prepend-label > span {
 /********************* MONSTER PANEL TEXT *******************/
 /************************************************************/
 .monster-panel-text {
-	box-shadow: 1px 1px 1px #aaa;
-	box-sizing: border-box;
-	border-left-width: 4px;
-	border-left-style: solid;
+	display: block;
+	margin-bottom: 1rem;
 	width: 320px;
-}
+	padding: 0;
+	background: $panel-background-color;
+	border-radius: 2px 2px 2px 2px;
```

When you have the same value for all four directions, repeating it is superfluous.
```suggestion
	border-radius: 2px;
```

---

Did not noticed it 😅 Thanks!